client/components/telecom/telephony/associateDevice/telecom-telephony-associate-device.html

```diff
@@ -6,47 +6,50 @@ <h2 data-translate="loading"></h2>
 
      <div class="alert alert-success"
           role="alert"
-          data-ng-if="$ctrl.attachSuccess && $ctrl.isInitialized"
-          data-translate="telephony_device_associate_attach_success">
+          data-ng-if="$ctrl.attachSuccess && $ctrl.isInitialized">
+          <p data-translate="telephony_device_associate_attach_success"></p>
      </div>
 
      <div class="alert alert-info"
           role="alert"
-          data-ng-if="!$ctrl.phones.length && $ctrl.isInitialized"
-          data-translate="telephony_device_associate_attach_empty">
+          data-ng-if="!$ctrl.phones.length && $ctrl.isInitialized">
+          <p data-translate="telephony_device_associate_attach_empty"></p>
      </div>
 
      <form data-ng-if="$ctrl.isInitialized && !$ctrl.attachSuccess && $ctrl.phones.length"
            data-ng-submit="$ctrl.attachDevice()">
-        <label class="block bottom-space-m12 pointer"
-               data-ng-repeat="phone in $ctrl.phones">
-            <flat-radio>
-                <input type="radio"
-                       data-ng-attr-id="{{ 'phone' + phone.macAddress }}"
-                       data-ng-attr-name="{{ 'phone' + phone.macAddress }}"
-                       data-ng-attr-value="{{ phone.macAddress }}"
-                       data-ng-model="$ctrl.selectedMac" />
-            </flat-radio>
-            <span class="left-space-m8"
-                  data-ng-bind="phone.description">
-            </span>
-            <span class="normal left-space-m16" data-ng-bind="phone.macAddress"></span>
-            <div class="left-space-m32">
-                <span class="normal"
-                      data-translate="telephony_device_associate_line_count"
-                      data-translate-values="{
-                          count: phone.associatedLines.length,
-                          total: phone.maxline
-                      }">
-                </span>
-                <ul class="normal left-space-m16">
-                    <li data-ng-repeat="line in phone.associatedLines">
-                        <span data-ng-bind="line.serviceName | phoneNumber"></span>
-                    </li>
-                </ul>
+
+        <div class="form-group">
+            <div data-ng-repeat="phone in $ctrl.phones">
+                <label class="d-block mb-3">
```

Comment:
Maybe you can remove `d-block mb-3` and directly wrap your label with a `<div class="radio">`